tests/cupy_tests/core_tests/test_ndarray.py

```diff
@@ -197,11 +199,14 @@ def test_copy_multi_device_non_contiguous(self, order):
     def test_copy_multi_device_non_contiguous_K(self):
         arr = _core.ndarray((20,))[::2]
         with cuda.Device(1):
-            with self.assertRaises(NotImplementedError):
+            with pytest.raises(NotImplementedError):
                 arr.copy('K')
 
     # See cupy/cupy#5004
     @testing.multi_gpu(2)
+    @pytest.mark.xfail(
+        runtime.is_hip,
+        reason='ROCm may work differently in async D2D copy with streams')
```

Comment:
Question: Is this new in ROCm 5.0?

Reply:
It must be no. I get failed in ROCm 4.3 on my local (with different values compared to that of ROCm 5.0). How about the recent result on Jenkins ROCm tests?

Reply:
Ah, Jenkins is not failing because it only has one GPU so this test was not running...